### PR TITLE
Add parameter selection to close mirrored service now ticket

### PR DESCRIPTION
**Pull request by YuvHayun**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [CIAC-3619](https://jira-hq.paloaltonetworks.local/browse/CIAC-3619)

## Description
Added support for the 'resolved' closing ticket option.

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

### CODE REVIEW

File: Packs/ServiceNow/ReleaseNotes/2_3_18.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 
 #### Playbooks
 ##### ServiceNow Ticket State Polling
```
Comment (JudahSchwartz) [inline Packs/ServiceNow/ReleaseNotes/2_3_18.md:3]: whats the change being made here? Can we revert changes to this file?

File: Packs/ServiceNow/ReleaseNotes/2_3_19.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### ServiceNow v2
+- Added the **close_ticket_multiple_options** integration param to replace **close_ticket** integration param. for further details please check the integration params table under **notes** section in the README file.
```
Comment (JudahSchwartz) [inline Packs/ServiceNow/ReleaseNotes/2_3_19.md:3]:
```suggestion
- Added the **close_ticket_multiple_options** integration param to replace **close_ticket** integration param. For further details please check the integration params table under **notes** section in the README file.
```

File: Packs/ServiceNow/Integrations/ServiceNowv2/README.md [hunk 3]
```diff
@@ -94,6 +94,9 @@ If MFA is enabled for your user, follow the next steps:
 | update_timestamp_field | Timestamp field to query for updates as part of the mirroring flow. | False |
 | mirror_limit | The maximum number of incidents to mirror incoming each time | False |
 | close_incident | Close XSOAR Incident. When selected, closing the ServiceNow ticket is mirrored in Cortex XSOAR. | False |
+| close_ticket_multiple_options | Define the closing method for XSOAR Incident. When selected, closing / resolving the ServiceNow ticket is mirrored in Cortex XSOAR. /
```
Comment (JudahSchwartz) [inline Packs/ServiceNow/Integrations/ServiceNowv2/README.md:97]: can we make the decription here the same as in the yml?

**Review by JudahSchwartz — COMMENTED** [review]
Looks great. Lets change the dexcription in readme to match the yml

File: Packs/ServiceNow/Integrations/ServiceNowv2/README.md (hunk 3 above)
Comment (YuvHayun) [reply Packs/ServiceNow/Integrations/ServiceNowv2/README.md:97]: Bar asked me to put more detailed information about pre-configured integrations somewhere.
Is there anywhere else you suggest putting this info?

Comment (JudahSchwartz) [reply Packs/ServiceNow/Integrations/ServiceNowv2/README.md:97]: I think releasenotes would probably be a good place then

**Review by JudahSchwartz — APPROVED** [review]

**Pull request merged**